### PR TITLE
Add vehicle locations to main

### PR DESCRIPTION
**Pull request by julyto**

### CODE REVIEW

File: internal/vehiclelocations/vehiclelocations.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func (d *VehicleLocations) UpdateVehicleLocation(vehicleGtfsRt VehicleGtfsRt, lo
 	d.vehicleLocations[idGtfsrt].Longitude = vehicleGtfsRt.Longitude
 	d.vehicleLocations[idGtfsrt].Bearing = vehicleGtfsRt.Bearing
 	d.vehicleLocations[idGtfsrt].Speed = vehicleGtfsRt.Speed
-	logrus.Debug("*** Vehicle Location updated: ", vehicleGtfsRt.Trip)
+	//logrus.Debug("*** Vehicle Location updated: ", vehicleGtfsRt.Trip)
```
Comment (benoit-bst) [inline internal/vehiclelocations/vehiclelocations.go:66]: useless ?

**Review by benoit-bst — APPROVED** [review]

File: cmd/forseti/main.go [hunk 2]
```diff
@@ -97,6 +111,19 @@ func GetConfig() (Config, error) {
 	pflag.Duration("routeschedule-refresh", 24*time.Hour, "time between refresh of RouteSchedules from navitia")
 	pflag.Duration("occupancy-clean-vj", 24*time.Hour, "time between clean list of VehicleJourneys")
 	pflag.Duration("occupancy-clean-vo", 2*time.Hour, "time between clean list of VehicleOccupancies")
+
+	//Passing configurations for vehicle_locations
+	pflag.String("locations-files-uri", "", "format: [scheme:][//[userinfo@]host][/]path")
```
Comment (kadhikari) [inline cmd/forseti/main.go:116]: What we use it for ?

**Review by kadhikari — CHANGES_REQUESTED** [review]

File: internal/vehiclelocations/vehiclelocations.go (hunk 1 above)
Comment (julyto) [reply internal/vehiclelocations/vehiclelocations.go:66]: corrected

File: cmd/forseti/main.go [hunk 3]
```diff
@@ -279,6 +312,37 @@ func VehiculeOccupancies(manager *manager.DataManager, config *Config, router *g
 	}
 }
 
+func VehiculeLocations(manager *manager.DataManager, config *Config, router *gin.Engine, location *time.Location) {
```
[the remark this answers is not in the document]
Comment (julyto) [reply cmd/forseti/main.go:315]: corrected

File: cmd/forseti/main.go [hunk 4]
```diff
@@ -279,6 +312,37 @@ func VehiculeOccupancies(manager *manager.DataManager, config *Config, router *g
...
+	if len(config.LocationsNavitiaURI.String()) == 0 || len(config.LocationsServiceURI.String()) == 0 {
+		logrus.Debug("Vehicle locations is disabled")
+		return
+	}
+
+	var vehiculeLocationsContext vehiclelocations.IConnectors
```
[the remark this answers is not in the document]
Comment (julyto) [reply cmd/forseti/main.go:321]: corrected and verify everywhere

File: internal/vehiclelocations/vehiclelocations.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (julyto) [reply internal/vehiclelocations/vehiclelocations.go:66]: corrected

File: cmd/forseti/main.go (hunk 2 above)
Comment (julyto) [reply cmd/forseti/main.go:116]: I keep this parameter as in Vehicle_Occupancy, but we usen't that at this time.

**Review by kadhikari — APPROVED** [review]

**Pull request merged**